src/main/java/no/publishers/model/PublisherDB.java

```diff
@@ -14,7 +14,9 @@
     private String uri;
     private String organizationId;
     private String orgPath;
-    private String prefLabel;
+    private String nbPrefLabel;
```

Comment:
Bør ikke dette heller vaære en HashMap<String,String> med språkkode som nøkkel?

Reply:
Sannsynligvis, regner egentlig med at jeg kan bruke prefLabel uten å mappe den om i det hele tatt. Men gjorde bare noe jeg visste fungerte med både graphql og mongo nå med en gang. 

Kommer en pr for create-løpet senere, ser nærmere på dette da.